fe/fe-core/src/main/java/com/starrocks/catalog/WorkGroupMgr.java

```diff
@@ -119,8 +119,11 @@ public void createWorkGroup(CreateWorkGroupStmt stmt) throws DdlException {
                 //default_cluster:test
                 String[] userParts = qualifiedUser.split(":");
                 String user = userParts[userParts.length - 1];
-                String roleName = Catalog.getCurrentCatalog().getAuth()
+                String qualifiedRoleName = Catalog.getCurrentCatalog().getAuth()
                         .getRoleName(ConnectContext.get().getCurrentUserIdentity());
+                //default_cluster:role
+                String[] roleParts = qualifiedRoleName.split(":");
+                String roleName = roleParts[roleParts.length - 1];
```

Comment:
here could throw  ArrayIndexOutOfBoundsException?

Reply:
NPE may be thrown, 
ArrayIndexOutOfBoundsException is impossible;
for an example:
1. when qualifiedRoleName="default_cluster:role1", split  into two parts, so roleParts.length-1 = 1; safe
2.  when qualifiedRoleName="role1", split into one part, roleParts.length-1 = 0; safe
3. qualifiedRoleName maybe null, so throw NPE.